ios/Framework/Core/Auth/UserPortraitScreenlet/Interactors/DownloadUserPortraitInteractor.swift

```diff
@@ -243,9 +243,9 @@ class DownloadUserPortraitInteractor: ServerReadConnectorInteractor {
 		let portraitEntry = attributes?["portraitId"]
 		let userEntry = attributes?["userId"]
 		if let attributes = attributes,
-				portraitId = portraitEntry?.description.asLong,
+				portraitId = portraitEntry?.longLongValue,
```

Comment:
if `asLong()` doesn't make sense anymore, just remove it and use your solution everywhere